### PR TITLE
Homework

### PR DESCRIPTION
**Pull request by Eukst**
Divese implementation & pg setup

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -76,7 +77,7 @@ group :test do
   gem 'webdrivers'
 end
 group :production do
-gem 'pg','1.1.4'
+  gem 'pg'
```
Comment (Sharee-ongraph) [inline Gemfile]: no need to add separately for production

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::Base
```
Comment (Sharee-ongraph) [inline app/controllers/application_controller.rb:4]: comment for class and methods

File: Gemfile (hunk 1 above)
Comment (Eukst) [reply Gemfile]: done

File: app/controllers/application_controller.rb (hunk 2 above)
Comment (Eukst) [reply app/controllers/application_controller.rb:4]: done

**Review by mitaligaur — APPROVED** [review]

**Review by Sharee-ongraph — APPROVED** [review]

**Pull request merged**